Service/NameConverter/HyphenToUnderscoreNameConverterDecorator.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types=1);
+
+namespace Auto1\ServiceAPIComponentsBundle\Service\NameConverter;
+
+use Symfony\Component\Serializer\NameConverter\NameConverterInterface;
+use function \str_replace;
```

Comment:
@sklabinsky what's the coding standard should we use for built-in function, explicit import or just prefixing with `\` or even skip the prefixing?

Reply:
Unfortunately we do not have strict requirements regarding this question.
But **normally** we do not `use function` - we prefix it.

Reply:
@sklabinsky @egorkrestinin1 There are no strict rules for using import for global function now. But one year ago, Doctrine Project, that has strict code style rules, begin strictly use import instead full qualified function names. I guess, to increase code readability.